src/SDDPoptimize.jl

```diff
@@ -40,6 +40,10 @@ function solve_SDDP(model::SPModel, param::SDDPparameters, verbose=0::Int64)
     if model.IS_SMIP && isa(param.MIPSOLVER, Void)
```

Comment:
is it possible to make a method param.test() ? It would be easier than to keep track of incompatibilities in the main code.